JS-Algorithms/challenges.js

```diff
@@ -71,6 +81,14 @@ The last 3 digits for the sum of powers from 1 to 10 is "317"
 
 const ownPower = (number, lastDigits) => {
   // YOUR CODE HERE...
+  let power = 0;
+  for (let i = 1; i <= number; i++) {
+      power += i**i;
+  }
+  power = BigInt(power);
+  power = power.toString();
+  power = power.substring(power.length-lastDigits,power.length);
```

Comment:
Avoid overwriting values on the same variable, use another one to save intermediate processes